xbmc/dialogs/GUIDialogYesNo.h

```diff
@@ -33,6 +33,11 @@ class CGUIDialogYesNo :
   virtual bool OnMessage(CGUIMessage& message);
   virtual bool OnBack(int actionID);
 
+  enum TimeOut
+  {
```

Comment:
Not in this case because a class enum doesn't cast to int and we want it to have the value 0